### PR TITLE
Provide active state on repository folders tree items.

### PR DESCRIPTION
**Pull request by bierik**
Closes https://github.com/4teamwork/opengever.core/issues/2875

- Enrich node data with active state

screen shot 2017-06-16 at 16 42 31

### CODE REVIEW

**Comment by lukasgraf** [comment]
@bierik tests are failing

**Comment by lukasgraf** [comment]
Test failures in `master` fixed via #3039

**Comment by lukasgraf** [comment]
Rebased on behalf of @bierik

File: opengever/base/browser/navigation.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from AccessControl import getSecurityManager
 from opengever.base.utils import get_preferred_language_code
+from opengever.repository.repositoryfolder import DOSSIER_STATES_INACTIVE
```
Comment (phgross) [inline opengever/base/browser/navigation.py]: DOSSIER_STATES_INACTIVE 🤔

File: opengever/base/browser/navigation.py [hunk 2]
```diff
@@ -88,7 +96,8 @@ def _brain_to_node(self, brain):
         return {'text': brain.Title,
                 'description': brain.Description,
                 'url': brain.getURL(),
-                'uid': brain.UID}
+                'uid': brain.UID,
+                'active': brain.review_state != DOSSIER_STATES_INACTIVE, }
```
Comment (phgross) [inline opengever/base/browser/navigation.py]: DOSSIER_STATES_INACTIVE 🤔

File: opengever/repository/repositoryfolder.py [hunk 3]
```diff
@@ -16,6 +16,8 @@
 from zope import schema
 from zope.interface import implements
 
+DOSSIER_STATES_INACTIVE = 'repositoryfolder-state-inactive'
```
Comment (phgross) [inline opengever/repository/repositoryfolder.py]: DOSSIER_STATES_INACTIVE 🤔

File: opengever/base/browser/navigation.py [hunk 4]
```diff
@@ -78,6 +79,13 @@ def query(self):
                 'path': '/'.join(self.context.getPhysicalPath()),
                 'sort_on': 'sortable_title'}
 
+    def is_active_node(self, node, inactive_nodes):
```
Comment (phgross) [inline opengever/base/browser/navigation.py]: `is_active_node` ist not used, leftover?

File: opengever/base/browser/navigation.py [hunk 5]
```diff
@@ -78,6 +79,13 @@ def query(self):
...
+        return node not in inactive_nodes
+
+    def _mark_active_node(self, node, inactive_nodes):
```
Comment (phgross) [inline opengever/base/browser/navigation.py]: is not used, leftover of a previous implementation ?

File: opengever/dossier/indexers.py [hunk 6]
```diff
@@ -11,10 +11,12 @@
 from opengever.dossier.utils import get_main_dossier
 from opengever.inbox.inbox import IInbox
 from opengever.private.dossier import IPrivateDossier
+from opengever.repository.repositoryfolder import IRepositoryFolderSchema
```
Comment (phgross) [inline opengever/dossier/indexers.py]: ??? 🤔

File: opengever/dossier/indexers.py [hunk 7]
```diff
@@ -11,10 +11,12 @@
...
 from plone.dexterity.interfaces import IDexterityContent
 from plone.indexer import indexer
 from Products.CMFCore.interfaces import ISiteRoot
 from zope.component import getAdapter
+from zope.component import getMultiAdapter
```
Comment (phgross) [inline opengever/dossier/indexers.py]: 🤔

**Comment by bierik** [comment]
@phgross updated

**Pull request merged**